src/scripts.js

```diff
@@ -24,6 +20,61 @@ const filterButton = document.querySelector('#filterButton');
 const mealPlanButton = document.querySelector('#mealPlanButton');
 const sideBarModal = document.querySelector('#sideBarModal');
 
+const ingredients = ingredientData.ingredientsData;
+const recipes = recipeData.recipeData.reduce((sum, recipe) => {
+  sum.push(new Recipe(recipe));
+  return sum
+},[])
+const users = userData.usersData;
+const cookbook = new Cookbook(ingredients, recipes)
+
+// helper functions
+
+const updateInnerText = (element, update) => {
+    element.innerText = update;
+}
+
+const toggleClass = (remove, add) => {
+  removeClass(remove);
+  addClass(add);  
+}
+
+const removeClass = (elements, rule) => {
+    elements.forEach(item => item.classList.remove(rule));
+}
+
+const addClass = (elements, rule) => {
+    elements.forEach(item => item.classList.add(rule));
+}
+
+const displayCurrentRecipes = (currentRecipes) => {
+  let display = currentRecipes;
+  mainDisplay.innerHTML = ""
+  display.forEach(recipe => {
+    mainDisplay.innerHTML +=
+    `<article class="flex column sml-brdr-radius shadow">
```

Comment:
This is a beautiful piece of code. Going to merge it for now, but tomorrow lets talk about the functionality served by line 51, not sure I understand it. Great work!